services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -118,8 +114,32 @@ public void initStorage() {
     STDOUT.log(Level.DEBUG, "BeaconChainController.initStorage()");
     this.chainStorageClient = ChainStorage.Create(ChainStorageClient.class, eventBus);
     this.chainStorageClient.setGenesisTime(UnsignedLong.valueOf(config.getGenesisTime()));
-    BeaconStateWithCache initialState = StartupUtil.createInitialBeaconState(config);
-    this.chainStorageClient.setStore(InteropChainStorageServer.get_genesis_store(initialState));
+    final String startState = config.getStartState();
+    BeaconStateWithCache initialState;
```

Comment:
ok so this is basically saying if there is a startState file specified, then use it; otherwise, use MockStart

Reply:
yes, exactly, I didn't want to further specify anything about deposit mode until we cleared that out with Joe.